src/Components/CollectionsHubRails/ArtistSeriesRail/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, color, Serif } from "@artsy/palette"
+import { ArtistSeriesRail_collectionGroup } from "__generated__/ArtistSeriesRail_collectionGroup.graphql"
+import { ArrowButton, Carousel } from "Components/v2/Carousel"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { data as sd } from "sharify"
+import styled from "styled-components"
+import { ArtistSeriesRailContainer as ArtistSeriesEntity } from "./ArtistSeriesEntity"
+
+export interface ArtistSeriesRailProps {
+  collectionGroup: ArtistSeriesRail_collectionGroup
+}
+
+export const ArtistSeriesRail: React.FC<ArtistSeriesRailProps> = ({
+  collectionGroup,
+}) => {
+  const { members } = collectionGroup
+  return (
+    <Content mb={2}>
+      <Serif size="5">Trending Artist Series</Serif>
+      <Carousel
+        height="200px"
+        width="500px"
+        options={{
+          groupCells: sd.IS_MOBILE ? 1 : 4,
+          wrapAround: sd.IS_MOBILE ? true : false,
+          cellAlign: "left",
+          pageDots: false,
+          draggable: sd.IS_MOBILE ? true : false,
+        }}
+        data={members}
+        render={slide => {
+          return <ArtistSeriesEntity member={slide} />
+        }}
+        renderLeftArrow={({ Arrow }) => {
+          return (
+            <ArrowContainer>
+              <Arrow />
+            </ArrowContainer>
+          )
+        }}
+        renderRightArrow={({ Arrow }) => {
+          return (
+            <ArrowContainer>
+              <ArrowContainer>{members.length > 4 && <Arrow />}</ArrowContainer>
+            </ArrowContainer>
+          )
+        }}
+      />
+    </Content>
+  )
+}
+
+const Content = styled(Box)`
+  border: 1px solid ${color("black10")};
+  padding-left: 20px;
```

Comment:
With our design system [palette](https://github.com/artsy/palette), it's preferable to specify padding like this using the `px` (horizontal padding) and `py` (vertical padding) props on the `<Content>` declaration in the rendering of the component. 

This would mean removing lines 56-59 here, and modifying [line 19](https://github.com/artsy/reaction/pull/2644/files#diff-7146303d88d4a2a5cbd339d3eed6fcbdR19) to read like this: 

`<Content mb={2} px={2} py={3}>`.

We don't have great docs in Palette on what types of props are available for things like padding, but they come from a dependency named [`styled-system`](https://styled-system.com/api). Any of the props defined by `styled-system` are available on things like a `Box` or `Flex` component, and when we _can_ use them, we prefer that to specifying padding in a styled component like this.

One other trick to know is that when you _do_ specify something like `mb={2}` (margin-bottom), `px={2}` (horizontal padding), or `py={3}` (vertical padding), the values are generally changed by a factor of 10. So `px={2}` equates to `padding-left: 20px; padding-right: 20px`, and `py={3}` equates to `padding-top: 30px; padding-bottom: 30px`. These values are all based on our theme, [which defines all the different spacing values](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L116) (amongst other things).